external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaRelationSuite.scala

```diff
@@ -147,6 +148,9 @@ class KafkaRelationSuite extends QueryTest with BeforeAndAfter with SharedSQLCon
   }
 
   test("test late binding start offsets") {
+    // Kafka fails to remove the logs on Windows. See KAFKA-1194.
+    assume(!Utils.isWindows)
+
```

Comment:
The exception is raised in https://github.com/apache/spark/pull/16999/files/0dd78c159ef9cff2a711ffb7f182ce832ef9cf18#diff-ed9a2655a6ce82a33879b159863738d1R176